build-logic-settings/build-environment/src/main/kotlin/gradlebuild/basics/BuildEnvironmentService.kt

```diff
@@ -44,7 +55,9 @@ abstract class BuildEnvironmentService : BuildService<BuildEnvironmentService.Pa
         val projectDir = parameters.rootProjectDir.asFile.get()
         val execOutput = providers.exec {
             workingDir = projectDir
-            isIgnoreExitValue = true
+            DeprecationLogger.whileDisabled {
+                isIgnoreExitValue = true
+            }
```

Comment:
❔ This is needed because of the `is`-prefix, right?

Reply:
Seconded, can't this use `ignoreExitValue = true`?

Reply:
>❔ This is needed because of the is-prefix, right?

Yes that is the case

> Seconded, can't this use ignoreExitValue = true?

We can't use `ignoreExitValue = true` since this is build-logic code and it has to compile with Gradle 8 and Gradle 9 (for Gradleception). That is why we still use `isIgnoreExitValue` for now